src/ts/app/view/reactbridge.ts

```diff
@@ -153,7 +153,7 @@ export class ReactBridge {
 
     renderHelp() {
         const props: HelpProps = {
-            isVisable: this.app.isHelpOverlayOn,
+            isVisible: this.app.isHelpOverlayOn,
             onClick: () => this.app.toggleHelpOverlay()
         }
         const help = Help(props)
```

Comment:
it seems there should be more extensive change to `reactbridge` to integrate the new React Modal (doesn't seem to currently be used?)